ArchiSteamFarm/Bot.cs

```diff
@@ -1026,6 +1026,22 @@ internal sealed class Bot : IDisposable {
 								return await ResponseAdvancedRedeem(steamID, args[1], args[2]).ConfigureAwait(false);
 							}
 
+							goto default;
+						case "!TAKE":
+							if (args.Length > 2) {
+								return await ResponseTake(steamID, Utilities.GetArgsString(args, 1, ","), args[2]).ConfigureAwait(false);
```

Comment:
This is wrong on so many levels - `Utilities.GetArgsString()` should always point to the last argument, not any argument in the middle.